packages/calcite-components/src/components/shell-panel/shell-panel.scss

```diff
@@ -123,6 +123,25 @@
   }
 }
 
+.float-all {
+  max-block-size: var(--calcite-internal-shell-panel-max-height, calc(100% - 1rem));
+  box-shadow: var(--calcite-shadow-sm);
+  border-radius: var(--calcite-corner-radius-round);
```

Comment:
Does this need a token for customization?

Reply:
I see this happen in Popover and also in Dialog (see the top corners)
![CleanShot 2025-01-30 at 11 45 38@2x](https://github.com/user-attachments/assets/e819ef59-dad0-43a5-929f-17c6549e840a)

Spoke to @jcfranco about this a while ago and he had similar concerns you outline in your second bullet, but also thought it could be worth looking into. Might adversely affect floating ui slotted into a Dialog...

Reply:
Yeah, we could approach it in a few ways, all have pros / cons. For now, I think I'll leave this as-is, since it's not unique to Shell Panel, and should be done in a way that addresses all patterns like this.

Reply:
@jcfranco ok to proceed for now?